### ISSUE TITLE
High DPI strikes again: Incorrect axis positions on dual-monitors with different resolution

### ISSUE THREAD

**Issue by ntjess** [pyqtgraph/pyqtgraph#1250]
### Short description

Creating a plot widget when you have two monitors with different resolutions causes the axes to be in the wrong locations / not scaled correctly with ImageItems.
### Code to reproduce

```python
import pyqtgraph as pg
from skimage import data


app = pg.mkQApp()
pg.setConfigOption('imageAxisOrder', 'row-major')
imgNp = data.chelsea()
imgItem = pg.ImageItem(image=imgNp)

plt = pg.PlotWindow()
plt.getViewBox().invertY(True)
plt.addItem(imgItem)
plt.show()
app.exec()
```

### Expected behavior

Axes should look the same on both monitors
### Real behavior

Axis scaling is correct on high-dpi monitor:
![image](https://user-images.githubusercontent.com/23620506/84523597-8c807980-aca6-11ea-9bb1-2e196d5de126.png)

Axis scaling is strange on the low-dpi monitor:
![image](https://user-images.githubusercontent.com/23620506/84523454-58a55400-aca6-11ea-9a16-ef71bf30712a.png)

### Tested environment(s)

 * PyQtGraph version: 0.11.0.dev0+g720fa5f
 * Qt Python binding: 5.13.0
 * Python version: 
 * NumPy version: 
 * Operating system: 
 * Installation method:

**Comment by ntjess** [comment]
This also manifests itself when making custom spot shapes on a high-dpi monitor and dragging the window to low-dpi monitor:
Spots on high dpi:
![image](https://user-images.githubusercontent.com/23620506/84523809-f567f180-aca6-11ea-8c6c-a5f2cf1eedd3.png)
Spots/scaling on low-dpi:
![image](https://user-images.githubusercontent.com/23620506/84523840-0153b380-aca7-11ea-9177-ec98c5a098a7.png)

**Comment by j9ac9k** [comment]
I now have a laptop with a HiDPI screen and an external display that isn't HiDPI, so I should probably start to look into these HiDPI issues.  Given the number of HiDPI related features in Qt being one of recent versions, it's unlikely that we could roll back fixes into the currently supported versions of Qt.

**Comment by j9ac9k** [comment]
Looks like issue will be fixed with #1509 


hidpi
![image](https://user-images.githubusercontent.com/646398/105614796-914ef500-5d80-11eb-8feb-beaafc340286.png)


lowdpi
![image](https://user-images.githubusercontent.com/646398/105614809-aaf03c80-5d80-11eb-94c3-b3eb98017f4c.png)


For added difficulty, on the hidpi monitor, I had 150% scaling
